pkg/utils/ioutil.go

```diff
@@ -148,3 +151,54 @@ func Move(src, dst string) error {
 	}
 	return errors.Trace(os.RemoveAll(src))
 }
+
+// CreateDir creates the directory if it not exists.
+func CreateDir(path string) error {
+	if _, err := os.Stat(path); err != nil {
+		if os.IsNotExist(err) {
+			return os.MkdirAll(path, 0755)
+		}
+		return err
+	}
+	return nil
+}
+
+// CopyFile copies a file from src to dst
+func CopyFile(src, dst string) error {
```

Comment:
Can we use `utils.Copy`, which can copy both directories and files.

Reply:
It's not related to the tispark integration, we could do that in a dedicated PR.